rules/windows/powershell/powershell_script/posh_ps_namespace_X509Certificates.yml

```diff
@@ -0,0 +1,29 @@
+title: PowerShell X509Certificate Namespace in ScriptBlock
+id: 34949d3a-0684-4300-9ab1-2a50daf920fb
+status: experimental
+description: Detecting Hoaxshell reverse-shell (HTTPS Payload) which is using System.Security.Cryptography.X509Certificates namespace in the payload.
```

Comment:
The description doesn't reflect the title of the rule.  Consider changing the title to something about `Hoaxshell` or the description